readme.md

```diff
@@ -36,8 +36,8 @@ Prowadzący następnego spotkania powinien być wytypowany na bieżącym spotkan
 | #28 | 15 maja 2023         |                                                      |                               |                                                                                                              |
 | #29 | 20 czerwca 2023      | [Adrian Gawle](https://github.com/agawle)            | Server-sent events            | [link](https://github.com/blumilksoftware/internal-meetups/tree/main/2023-06-20-server-sent-events)          |
 | #30 | 17 lipca 2023        |                                                      |                               |                                                                                                              |
```

Comment:
I think yours should be listed as number 30.

Reply:
I think 30 number is number of @nadybski presentation.

Reply:
Updated